app/ui/browser2/views/browser-content.jsx

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Yeah, I guess this should pretty much be used for any kind of traditional browser chrome